daml-lf/transaction/src/main/scala/com/daml/lf/Error.scala

```diff
@@ -104,4 +104,6 @@ object Error {
 
   final case class ContractIdInContractKey(key: Value[ContractId]) extends Error
 
+  final case object TooDeepValue extends Error
```

Comment:
```suggestion
  final case object ValueExceedsMaxNesting extends Error
```
sounds a bit clearer but I don’t feel particularly strongly about this.